layouts/header.html

```diff
@@ -7,10 +7,10 @@
 
       <nav id="navigation">
         <ul>
```

Comment:
We're going to remove all these links in the Header. Update the code image to point to the homepage of code.cerner.com instead.